public/js/controllers/mempool.js

```diff
@@ -176,6 +184,42 @@
                 addTxRow(tx, this[tx.Type.toLowerCase() + "TransactionsTarget"], rowFn)
             })
         }
-    })
 
+        filterVotesLastBlock() {
```

Comment:
How about renaming this to something more semantic like `labelVotes`? 
The word "filter" implies creating a subset to me.

Reply:
or `categorizeVotes` or ...`classifyVotes`, too many good names to chose!

Reply:
That sounds like a good idea.  i'll update that later today.